blips/blip-0001.mediawiki

```diff
@@ -0,0 +1,242 @@
+
+For Lightning developers, bLIPs are a convenient way to track the progress of their implementation. Ideally, each implementation editor would list the bLIPs they have implemented. This will give end users a convenient way to know the current status of a given implementation or library.
+
+==bLIP Types==
+
+There are three kinds of bLIPs:
+
+* A Standards Track bLIP describes any change that affects most or all Lightning implementations, such as—a change to the network protocol, a change in HTLC types or funding transaction rules, proposed application standards/conventions, or any change or addition that affects the interoperability of applications using Lightning. Standards Track bLIPs consist of three parts—a design document, an implementation, and (if warranted) an update to the Basics of Lightning Technology (BOLT) specification. Furthermore, Standards Track bLIPs can be broken down into the following categories:
+** Core: improvements requiring changes to the BOLTs, as well as changes that are not necessarily critical but may be relevant to “protocol dev” discussions, and routing node strategy changes.
+** Routing/Pathfinding: includes improvements around the routing protocol, as well as proposed improvements to the onion routing protocol specification ([https://github.com/lightningnetwork/lightning-rfc/blob/master/04-onion-routing.md BOLT #4]).
+** Networking: includes improvements around the gossip protocol, as well as proposed improvements to network protocol specifications ([https://github.com/lightningnetwork/lightning-rfc/blob/master/07-routing-gossip.md BOLT #7]).
+** Lightning Request for Comments (LRC): application-level standards and conventions.
+* A Process bLIP describes a process surrounding Lightning, or proposes a change to (or an event in) a process. Process bLIPs are like Standards Track bLIPs but apply to areas other than the Lightning protocol itself. They may propose an implementation, but not to the BOLT specification; they often require community consensus; unlike Informational bLIPs, they are more than recommendations, and users are strongly encouraged to adopt them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Lightning development. Any meta-bLIP is also considered a Process bLIP.
+* An Informational bLIP describes a Lightning design issue, or provides general guidelines or information to the Lightning community, but does not propose a new feature. Informational bLIPs do not necessarily represent a Bitcoin community consensus or recommendation, so users and implementers are free to ignore Informational bLIPs or follow their advice.
```

Comment:
Hmm, is it the right place to put "here's one idea for how you can use lightning" right next to "here's a way we implement optional lightning features"?

Reply:
If I'm following correctly, you're saying Informational bLIPs ("here's one idea for how you can use lightning") are not desirable, instead this section should just describe "Process bLIPs" ("here's a way we implement optional lightning features"), yeah? 

I don't feel strongly about that one way or another, and maybe the "Process bLIP" sub-categories can be the three namespaces @ariard notes in the below comment?